run_build_impl.sh

```diff
@@ -191,6 +193,23 @@ else
   done
 fi
 
+# Automatic ground truth evaluation.
+if RUN_AUTOMATIC_EVALUATION; then
+  # Rotary dataset.
+  path_to_automatic_evaluation_repo=$WORKSPACE/$path_to_automatic_evaluation_repo
+  path_to_datasets=$path_to_automatic_evaluation_repo/$path_to_rotary_data
+  path_to_evaluation=$WORKSPACE/$path_to_python_evaluation_script
+  # The result directory consists of the current date and time.
+  output=$((python $path_to_evaluation/run_evaluation.py --job ci_jobs/job_euroc.yaml --dataset_path $path_to_datasets) 2>&1)
+  result_directory=$(echo $output | awk '{ print $(NF) }')
+  # Move the evaluation to the automatic evaluation repository and upload the results.
+  mv $result_directory $path_to_automatic_evaluation_repo/$path_to_rotary_results
+  cd $path_to_automatic_evaluation_repo
+  git add .
+  git commit -m "New results from automatic evaluation (rotary)."
+  git push origin master
```

Comment:
I think it would be better to move this evaluation scheme- specific logic to a script in https://github.com/ethz-asl/multiagent_mapping_automatic_evaluation and call that script from here.